main.go

```diff
@@ -141,6 +142,9 @@ func main() {
 	}
 
 	components.submSpec.CableDriver = strings.ToLower(components.submSpec.CableDriver)
+	if components.submSpec.CableDriver == vxlan.CableDriverName && components.submSpec.NATEnabled {
+		logger.Warning("VxLan cable-driver is supported only with no NAT deployments")
+	}
```

Comment:
I'd prefer to do this check in the [vxlan cable driver](https://github.com/submariner-io/submariner/blob/eafa40319d27327460351f9e107a351d0d9d6225/pkg/cable/vxlan/vxlan.go#L99).

Reply:
Thanks for reviewing!  Updated.